test/helpers/dummyPlugin.js

```diff
@@ -0,0 +1,45 @@
+function DummyPlugin (config) {
```

Comment:
What do you think about shipping this with index files for a couple of our most common plugins (maybe that use env vars to include config values like the XRP secret)?

Reply:
Sure, or maybe one env var 'PLUGIN_MODULE_NAME', and one 'CONFIG_JSON'?
I was thinking of adding various examples (also the ones currently included in the readme) into an `examples/` folder.

Reply:
CONFIG_JSON is pretty annoying to use as an env var so I think it would be slightly better to make plugin-specific env vars that it looks for

Also, nitpick but I think PLUGIN_MODULE is slightly better

Reply:
Done! https://github.com/interledgerjs/btp-toolbox/blob/f9a46d4dbb27614398a2d73d3cee93b98d4eb107/examples/frogBellsProxy.js